Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -181,6 +182,30 @@ def test_kill(self):
         else:
             self.assertEqual(-signal.SIGKILL, returncode)
 
+    def test_kill_issue43884(self):
+        blocking_shell_command = f'{sys.executable} -c "import time; time.sleep(100000000)"'
```

Comment:
```suggestion
        blocking_shell_command = f'{sys.executable} -c "import time; time.sleep(100000)"'
```

Reply:
@gvanrossum I think this should fix it. Can you commit this for me?

Reply:
3.10 has a bug where it does not properly casts sleep time leading to overflow which is fixed on 3.11 and main.